conf/application.conf

```diff
@@ -1,12 +1,12 @@
 include "common.conf"
 
-appName="address-lookup-frontend"
-application.router=prod.Routes
+appName = "address-lookup-frontend"
```

Comment:
Please do not introduce formatting changes in this PR, no matter how offensive the existing formatting ;-)